SourceX/controls/devices/joystick.cpp

```diff
@@ -225,7 +225,14 @@ bool Joystick::ProcessAxisMotion(const SDL_Event &event)
 #endif
 #ifdef JOY_AXIS_LEFTY
 	case JOY_AXIS_LEFTY:
-		leftStickYUnscaled = -event.jaxis.value;
+		// Work around Nimbus that reports inverted Y. No known fix,
+		// AFAIK, and no SDL remapping.
+		// https://forums.libsdl.org/viewtopic.php?p=51409
+		if (nimbus_axis_hack_) {

```

Comment:
Maybe this could be a more general flag option for the ini. Something like: "Joystick Invert Y-Axis".

It could even use it as a feature :man_shrugging:

Reply:
I was not sure about the usefulness of such a feature outside of this specific vendor, so I'd like to open a discussion.

Reply:
Inverted YAxis might happen with other joysticks:
https://github.com/gabomdq/SDL_GameControllerDB/issues/162#issuecomment-355218426